src/mixins/nft.js

```diff
@@ -774,6 +774,10 @@ export default {
       await this.fetchNFTDisplayStateListByAddress(this.getAddress);
     },
     async collectNFT() {
+      if (!this.nftIsCollectable) {
+        this.$router.push({ query: { ...this.$route.query } });
```

Comment:
Wouldn't this cause duplicated navigation? potentially breaks the page

Reply:
Updated